ikarus/finiteelements/feresulttypes.hh

```diff
@@ -252,4 +252,19 @@ auto toString() {
 template <template <typename, int, int> class RT1, template <typename, int, int> class RT2>
 constexpr bool isSameResultType = std::is_same_v<Impl::DummyRT<RT1>, Impl::DummyRT<RT2>>;
 
+/**
+ * \brief Meta function to test whether a given ResultType is in a tuple of supported ResultTypes
+ * \tparam SupportedRT tuple type of supported RestultTypes (exported by the elements)
+ * \tparam RT ResultType template to be checked
+ */
+template <typename SupportedRT, template <typename, int, int> class RT>
+static consteval bool isSupportedResultType() {
```

Comment:
I would be in favor with a base class approach such as: https://godbolt.org/z/Yz9hh79eG.
But maybe this is just a matter of taste. The difference would be that we have (if we want) a member function `supportsResultType` to ask the element.

Reply:
I also thought about that, but then we would need to subclass the elments right?